Makefile

```diff
@@ -171,7 +171,6 @@ test:
 .PHONY: test-cov
 test-cov:
 	@./hack/test-cover.sh ./cmd/... ./extensions/... ./pkg/... ./plugin/...
-	$(MAKE) test-prometheus
```

Comment:
Can we rather exclude `test-prometheus` from the `verify-extended` rule so that `test-cov` stays similar to `test`?